### ISSUE TITLE
Negative value guard

### ISSUE THREAD

**Issue by haku956** [eldruin/veml6075-rs#2]
In situations there is barely, or no, UV light the driver could report negative values which do not make sense. In `read(...)`:
```
        let uva = f32::from(uva)
            - (self.calibration.uva_visible * f32::from(uvcomp1))
            - (self.calibration.uva_ir * f32::from(uvcomp2));
        let uvb = f32::from(uvb)
            - (self.calibration.uvb_visible * f32::from(uvcomp1))
            - (self.calibration.uvb_ir * f32::from(uvcomp2));
```

which may lead to a negative uva and uvb; i think the value should be clamped to zero since negative values do not make physical sense. This could also be done in client code.

**Comment by eldruin** [comment]
Good catch. This makes sense. I will implement this soon and make a new release.

**Comment by haku956** [comment]
It also happens for the first measurement after the sensor is started e.g. use integration time 800

uva=-96.9000015258789,uvb=-139.22000122070313,uv_index=-0.2511449456214905

**Comment by tacuna** [comment]
You could even consider to let it return a core::option::Option as it is effectively different from an actual 0.0 measurement.

Would be a breaking change though and quite niche use-case.
